scipy/optimize/tests/test__basinhopping.py

```diff
@@ -449,7 +447,14 @@ def func(x):
         x0 = -4
         limit = 50  # Constrain to func value >= 50
         con = {'type': 'ineq', 'fun': lambda x: func(x) - limit},
-        res = basinhopping(func, x0, 30, minimizer_kwargs={'constraints': con})
+        seed = np.random.RandomState(1234)
+        res = basinhopping(
+            func,
+            x0,
+            30,
+            seed=1234,
```

Comment:
```suggestion
            seed=seed,
```
?

Reply:
Using `seed` here ensures the test continues to work after I removed the use of `np.random.seed`, because `seed` still uses `np.random.RandomState`.

Reply:
Feel free to fix the lint error however you see fit.